changelog.adoc

```diff
@@ -2,6 +2,16 @@
 
 // scriv-insert-here
 
+== 3.39.0 (2025-10-15)
+
+Enhancements:
+
+* Text display of Timers via commands such as `globus timer show` now includes
```

Comment:
Recommending distinguishing between the Timers service (capitalized) vs a timer resource (lowercase).

```suggestion
* Text display of timers via commands such as `globus timer show` now includes
```